src/Checkbox/README.md

```diff
@@ -66,7 +66,8 @@ Custom validation:
 
 	  <Checkbox
 	  	label="Required Example"
-	  	validator={ function(checked){ return {valid: checked, message: "Checkbox is required"} } }
+	  	required="custom required message"
```

Comment:
Please change label to read "Custom Validation Example" and change the required prop to read "Custom Required Message".